source/features/dim-bots.tsx

```diff
@@ -9,6 +9,14 @@ const commitSelector = [
 	'.commit-author[href$="%5Bbot%5D"]',
 	'.commit-author[href$="renovate-bot"]',
 	'.commit-author[href$="scala-steward"]',
+	'.commit-author[href$="actions-user"]',
+	'.commit-author[href$="web-flow"]',
+	'.commit-author[href$="bors"]',
+	'.commit-author[href$="rust-lang"]',
+	'.commit-author[href$="rust-highfive"]',
+	'.commit-author[href$="snyk-bot"]',
+	'.commit-author[href$="imgbot"]',
+	'.commit-author[href$="octomerger"]',
```

Comment:
This will be better as a map like <code>['bot1', 'bot2'].map(bot => \`.commit-author[href$="${bot}"]\`)</code>

Reply:
I'll change to use a map instead 👍

Reply:
You should keep them separated in lines for diff's sake